keyboards/nuphy/air75_v2/ansi/info.json

```diff
@@ -0,0 +1,268 @@
+            {"matrix": [2, 10], "x": 105, "y": 20, "flags": 4},
+            {"matrix": [2, 11], "x": 115, "y": 20, "flags": 4},
+            {"matrix": [2, 12], "x": 125, "y": 20, "flags": 4},
+            {"matrix": [2, 13], "x": 135, "y": 20, "flags": 4},
+            {"matrix": [2, 16], "x": 150, "y": 20, "flags": 4},
+
+            {"matrix": [1, 15], "x": 150, "y": 30, "flags": 4},
+            {"matrix": [3, 13], "x": 127.5, "y": 30, "flags": 4},
+            {"matrix": [3, 11], "x": 117.5, "y": 30, "flags": 4},
+            {"matrix": [3, 10], "x": 107.5, "y": 30, "flags": 4},
+            {"matrix": [3, 9], "x": 97.5, "y": 30, "flags": 4},
+            {"matrix": [3, 8], "x": 87.5, "y": 30, "flags": 4},
+            {"matrix": [3, 7], "x": 77.5, "y": 30, "flags": 4},
+            {"matrix": [3, 6], "x": 67.5, "y": 30, "flags": 4},
+            {"matrix": [3, 5], "x": 57.5, "y": 30, "flags": 4},
+            {"matrix": [3, 4], "x": 47.5, "y": 30, "flags": 4},
+            {"matrix": [3, 3], "x": 37.5, "y": 30, "flags": 4},
+            {"matrix": [3, 2], "x": 27.5, "y": 30, "flags": 4},
+            {"matrix": [3, 1], "x": 17.5, "y": 30, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 30, "flags": 4},
+
+            {"matrix": [4, 0], "x": 0, "y": 40, "flags": 4},
+            {"matrix": [4, 2], "x": 22.5, "y": 40, "flags": 4},
+            {"matrix": [4, 3], "x": 32.5, "y": 40, "flags": 4},
+            {"matrix": [4, 4], "x": 42.5, "y": 40, "flags": 4},
+            {"matrix": [4, 5], "x": 52.5, "y": 40, "flags": 4},
+            {"matrix": [4, 6], "x": 62.5, "y": 40, "flags": 4},
+            {"matrix": [4, 7], "x": 72.5, "y": 40, "flags": 4},
+            {"matrix": [4, 8], "x": 82.5, "y": 40, "flags": 4},
+            {"matrix": [4, 9], "x": 92.5, "y": 40, "flags": 4},
+            {"matrix": [4, 10], "x": 102.5, "y": 40, "flags": 4},
+            {"matrix": [4, 11], "x": 112.5, "y": 40, "flags": 4},
+            {"matrix": [4, 13], "x": 122.5, "y": 40, "flags": 4},
+            {"matrix": [4, 14], "x": 140, "y": 40, "flags": 4},
+            {"matrix": [2, 15], "x": 150, "y": 40, "flags": 4},
+
+            {"matrix": [5, 15], "x": 150, "y": 50, "flags": 4},
+            {"matrix": [5, 14], "x": 140, "y": 50, "flags": 4},
+            {"matrix": [5, 13], "x": 130, "y": 50, "flags": 4},
+            {"matrix": [3, 14], "x": 120, "y": 50, "flags": 4},
+            {"matrix": [5, 10], "x": 110, "y": 50, "flags": 4},
+            {"matrix": [5, 9], "x": 100, "y": 50, "flags": 4},
+            {"matrix": [5, 6], "x": 37.5, "y": 50, "flags": 4},
+            {"matrix": [5, 2], "x": 25, "y": 50, "flags": 4},
+            {"matrix": [5, 1], "x": 12.5, "y": 50, "flags": 4},
+            {"matrix": [5, 0], "x": 0, "y": 50, "flags": 4},
+            {"flags": 4}
+        ]
+    },
+    "ws2812": {
+        "pin": "A7"
+    },
+    "layouts": {
+        "LAYOUT_ansi_84": {
```

Comment:
```suggestion
    },
    "community_layouts": ["75_ansi"],
    "layouts": {
        "LAYOUT_75_ansi": {
```

Reply:
This is a 84key keyboard, air75 is just a product name, Do I need change 84 to 75?

Reply:
The nomenclature of 75% is a catch-all for all other boards in this layout. Since this board has an F-row and the extra column of keys to the right above the Right Arrow key, it would fall into using `LAYOUT_75_ansi`.

Reply:
OK, thanks